### ISSUE TITLE
Markdown Preview does not transfer file to browser.

### ISSUE THREAD

**Issue by seralekseenko** [facelessuser/MarkdownPreview#115]
Greetings. :)

Thanks for the package! This is a very useful thing.

But I ran into a problem in its use.

When sending the `README.md` file to the browser, `file: ///tmp/13.html` opens in the browser, but "ERR_FILE_NOT_FOUND" is displayed on the page.

Operating System: Ubuntu 20.04 LTS.
Browser: Chromium v 83.0.4103.116

My attempts to find the reason in the settings of the sublime or the package did not lead to a positive result.

I already found a similar closed issue, but I did not consider the solution in it.

Any ideas?

**Comment by facelessuser** [comment]
I'm not sure what your Markdown Preview settings look like. Also, did you restart Sublime after installing Markdown Preview? Please provide more information. I currently have no issues with my machines. Granted, I haven't tested on Ubuntu 20.04 LTS yet, but this issue may not be specific to Ubuntu and may just be an environment issue. I just can't tell based on the info provided.

@gir-bot add S: more-info-needed

**Comment by seralekseenko** [comment]
Thanks for the answer. Your answer prompted me some thoughts about the package settings. And I decided to experiment.
Which, however, did not solve the problem that I wrote about in a previous post.

In the end, I came to a complete reinstall of Sublime. That everything would be clean.
I didn’t touch the settings at all, neither in Sublime nor in Markdown Preview.

**The result:** the problem was in the Chromium browser.

And with Mozilla and with Chrome everything is fine, there were no problems. This is the right solution for me. :)

Sorry to bother you on such a trifle.
